pkg/rbac/common.go

```diff
@@ -3,28 +3,51 @@ package rbac
 import (
 	"crypto/sha256"
 	"encoding/base32"
+	"fmt"
 	"strings"
 
 	"github.com/pkg/errors"
 	"github.com/rancher/norman/types"
 	"github.com/rancher/norman/types/slice"
 	mgmt "github.com/rancher/rancher/pkg/apis/management.cattle.io"
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
 	provv1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
 	v32 "github.com/rancher/rancher/pkg/generated/controllers/management.cattle.io/v3"
-	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+	normanv3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
 	"github.com/rancher/rancher/pkg/ref"
+	"github.com/rancher/wrangler/pkg/name"
 	k8srbacv1 "github.com/rancher/wrangler/v3/pkg/generated/controllers/rbac/v1"
+	"github.com/rancher/wrangler/v3/pkg/generic"
 	wranglerName "github.com/rancher/wrangler/v3/pkg/name"
 	"github.com/sirupsen/logrus"
 	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 )
 
 const (
-	NamespaceID = "namespaceId"
-	ProjectID   = "projectId"
-	ClusterID   = "clusterId"
-	GlobalAdmin = "admin"
+	NamespaceID                       = "namespaceId"
+	ProjectID                         = "projectId"
+	ClusterID                         = "clusterId"
+	GlobalAdmin                       = "admin"
+	GlobalRestrictedAdmin             = "restricted-admin"
+	ClusterCRDsClusterRole            = "cluster-crd-clusterRole"
+	RestrictedAdminClusterRoleBinding = "restricted-admin-rb-cluster"
+	ProjectCRDsClusterRole            = "project-crd-clusterRole"
+	RestrictedAdminProjectRoleBinding = "restricted-admin-rb-project"
+	RestrictedAdminCRForClusters      = "restricted-admin-cr-clusters"
+	RestrictedAdminCRBForClusters     = "restricted-admin-crb-clusters"
+	CrtbOwnerLabel                    = "authz.cluster.cattle.io/crtb-owner"
+	PrtbOwnerLabel                    = "authz.cluster.cattle.io/prtb-owner"
+	aggregationLabel                  = "management.cattle.io/aggregates"
+	clusterRoleOwnerAnnotation        = "authz.cluster.cattle.io/clusterrole-owner"
+	aggregatorSuffix                  = "aggregator"
+	promotedSuffix                    = "promoted"
+	clusterManagementPlaneSuffix      = "cluster-mgmt"
+	projectManagementPlaneSuffix      = "project-mgmt"
+	RBACApiGroup                      = "rbac.authorization.k8s.io"
```

Comment:
This is already defined in [`k8s.io/api/rbac/v1`](https://pkg.go.dev/k8s.io/api/rbac/v1#pkg-constants) as `GroupName`